coalib/bearlib/abstractions/Linter.py

```diff
@@ -766,6 +768,11 @@ def linter(executable: str,
     ...     def create_arguments(filename, file, config_file):
     ...         return "--lint", filename
 
+    The class is also a subclass of the ``LinterClass`` class.
+
+    >>> issubclass(XLintBear, LinterClass)
+    True
+
```

Comment:
I would maybe put the documentation before the `Documentation: ...` part. Imo it doesn't fit here very well^^